suse/build/SPECS/jenkins.spec

```diff
@@ -71,10 +71,10 @@ rm -rf "%{buildroot}"
 %__install -D -m0755 "%{SOURCE7}" "%{buildroot}%{_datadir}/%{name}/migrate"
 
 %pre
-/usr/sbin/groupadd -r %{name} &>/dev/null || :
+/usr/bin/getent group %{name} &>/dev/null || /usr/sbin/groupadd -r %{name} &>/dev/null
```

Comment:
this is the recommended method for adding a group depending on if it exists or not